### PR TITLE
Add a basic PostCSS setup

### PR DESCRIPTION
**Pull request by m-vo**
This adds a simple postcss chain for backend styles. The only plugin for now is "postcss-nested". Styles can now be added to the `assets/styles/backend.pcss` file. 

/cc @zoglo

### CODE REVIEW

File: core-bundle/assets/backend.js [hunk 1]
```diff
@@ -50,3 +50,6 @@ document.documentElement.addEventListener('turbo:frame-missing', (e) => {
     e.preventDefault();
     e.detail.visit(e.detail.response);
 });
+
+/* Styles */
```
Comment (leofeyer) [inline core-bundle/assets/backend.js]:
```suggestion
// Styles
```

File: core-bundle/assets/backend.js [hunk 2]
```diff
@@ -50,3 +50,6 @@ document.documentElement.addEventListener('turbo:frame-missing', (e) => {
...
+import './styles/backend.pcss';
```
Comment (leofeyer) [inline core-bundle/assets/backend.js]: Is it technically necessary that the import is at the end of the file? If not, I would prefer it to be where the other imports are.

File: webpack.config.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ Encore
     .disableSingleRuntimeChunk()
     .enableSourceMaps(!Encore.isProduction())
     .enableVersioning(Encore.isProduction())
+    .enablePostCssLoader()
```
Comment (leofeyer) [inline webpack.config.js]: Isn‘t there a `configurePostCssLoader()` method that makes adding a `postcss.config.js` file redundant?

File: core-bundle/assets/backend.js (hunk 1 above)
Comment (m-vo) [reply core-bundle/assets/backend.js]: I removed the comment in 230eabb7adbea0e91804553128b11ddd6ab03d02.

File: core-bundle/assets/backend.js (hunk 2 above)
Comment (m-vo) [reply core-bundle/assets/backend.js]: Moved to the top in 230eabb7adbea0e91804553128b11ddd6ab03d02.

File: webpack.config.js (hunk 3 above)
Comment (m-vo) [reply webpack.config.js]: Done in e8ec9fb5ef5f31938ff8a801c4d6ad504dd3da40.

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @m-vo.

**Comment by aschempp** [comment]
what do we add to this new file now, and do we add anything to the old files? What about new file that are included in backend.css, is that also possible (similar to SCSS)?

**Pull request merged**